src/DOC.md

```diff
@@ -0,0 +1,259 @@
+Beautiful, minimal, opinionated CLI prompts inspired by the
```

Comment:
I mean, need to revert this refactoring in this PR fully, it's another topic to consider, but it's not about multiline input